### DISCUSSION TITLE
Cloudron Update Not Appear

### DISCUSSION THREAD

**Discussion by Leandroledledled** [chatwoot/chatwoot#6547] [Self hosted]
Hello, I'm using Chatwoot on Cloudron.
Will this update be released to Cloudron automatically? Or do I need to install manual via terminal?
When I click to update it says it's already on the latest version. Help me please!

![Screenshot_16](https://user-images.githubusercontent.com/126264017/221390763-7b2cadaa-0b03-4675-abf9-7b1ce3c51c17.jpg)

**Comment by vishnu-narayanan** [comment]
@Leandroledledled Cloudron packages are released separately, and you should receive an option to update once that is out. Have you tried reaching out to Cloudron support forum?
